static/sass/main.scss

```diff
@@ -16,3 +16,7 @@
 @include multipass-github-buttons;
 @include multipass-p-strip;
 @include multipass-p-tabs;
+
+.p-navigation__banner {
+  padding-left: 0;
```

Comment:
I don't think it's needed anymore with proper use of `p-navigation__row--25-75`, as we already set it in Vanilla:

<img width="763" alt="image" src="https://github.com/canonical/multipass.run/assets/83575/66126b7b-e00d-424b-ab8b-88e111333176">

Reply:
And it actually causes a bug on docs layout. This should be removed, so the padding 0 is only applied when needed (by Vanilla):

<img width="837" alt="image" src="https://github.com/canonical/multipass.run/assets/83575/3cf36fc5-d004-4ab5-b2bf-5cfd4878d566">